web/modules/custom/dpl_webmaster/src/Form/InstallOrUpdateModule.php

```diff
@@ -198,6 +198,10 @@ public function submitForm(array &$form, FormStateInterface $form_state): void {
     // MODULE/) and others list an actual file (i.e., MODULE/README.TXT).
     $project = strtok($files[0], '/\\');
 
+    if (!$project) {
+      throw new \RuntimeException('Could not determine module name from archive');
+    }
```

Comment:
Please reconsider your error handling.

I appreciate the addition. Other code in this function handles error reporting by adding an error to messenger and then returning void.